examples/tpm2-extendpcr/extendpcr.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2018, Ian Haken. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Extends a pcr with some data.
+package main
+
+import (
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+var (
+	tpmPath = flag.String(
```

Comment:
The style in this repo (and more generally in go), is to keep these flag calls on a single line.

Reply:
Indented lines that wrap interrupt the visual structural flow of the file. For example

{
    this long line has to wrap and 
confuse the indentation
    this short line is fine
}

Also, repeated definitions that share similar layout are easier to parse, which is why I've formatted flag declarations that wouldn't wrap in the multi-line fashion as well.

I agree that keeping with existing style is important, though these are new files which I think creates enough distance for this change to be OK.

Reply:
I'm not sure I'm parsing the beginning of your comment well. Can you please rephrase?

My point is that it's the style of this repo (and most other Go code that I've written and read) to keep calls to flag.* on a single line. Please follow that style here.

Reply:
Oops, I meant to format that:
```
{
    this long line has to wrap and
confuse the indentation
    this short line is fine
}
```